frontend/src/components/common/Calendar/CheckInInfoCard.tsx

```diff
@@ -0,0 +1,238 @@
+import {
```

Comment:
Can we keep this file in `pages/AdminDashboard/components` to keep the same directory structure as DropoffCard?